app/assets/javascripts/catalog_manager/form.js.coffee

```diff
@@ -84,6 +84,15 @@ $ ->
       type: 'PUT'
       url: "/catalog_manager/super_user?super_user[identity_id]=#{identity_id}&super_user[organization_id]=#{organization_id}&super_user[access_empty_protocols]=#{access_empty_protocols}"
 
+  $(document).on 'change', '.su-allow-credit', ->
+    identity_id = $(this).data('identity-id')
+    organization_id = $(this).data('organization-id')
+    allow_credit = $(this).prop('checked')
+
+    $.ajax
+      type: 'PUT'
+      url: "/catalog_manager/super_user?super_user[identity_id]=#{identity_id}&super_user[organization_id]=#{organization_id}&super_user[allow_credit]=#{allow_credit}"
```

Comment:
Instead of nesting the data in the URL, you could use the AJAX `data` option to make it a little more readable.

```
$.ajax
  type: 'PUT'
  url: "/catalog_manager/super_user"
  data:
    super_user:
      identity_id: identity_id
      organization_id: organization_id
      allow_credit: allow_credit
```